.github/workflows/release-downloads-nuget.yml

```diff
@@ -118,3 +128,34 @@ jobs:
     - name: Check Java compile
       run: |
         dafny /compile:3 /spillTargetCode:3 /compileTarget:java a.dfy
+
+  test-dafny-libraries:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        library-name: [ DafnyPipeline, DafnyLanguageServer, DafnyRuntime, DafnyCore, DafnyDriver ]
```

Comment:
Should this include `DafnyServer`?

Not that we have to do it as part of this PR, but it would be great if there were a way to automatically populate this list.

Reply:
Yup, definitely, will add.

I made an attempt at automatically populating this, but it's a bit non-trivial since `dotnet install Dafny` never works, as it complains that you're supposed to `dotnet tool install` that instead, so I'd at least need to special-case that package. Added a comment to that effect too.